frontend/pages/admin/UserManagementPage/UserManagementPage.jsx

```diff
@@ -265,87 +240,62 @@ export class UserManagementPage extends Component {
     );
   }
 
-  renderUserTable = () => {
-    const { invites, users } = this.props;
-    const { renderUserRow } = this;
-
-    return (
-      <div className={`${baseClass}__wrapper`}>
-        <table className={`${baseClass}__table`}>
-          <thead>
-            <tr>
-              <th>Username</th>
-              <th>Status</th>
-              <th>Full Name</th>
-              <th>Email</th>
-              <th>Role</th>
-              <th className={`${baseClass}__position`}>Position</th>
-              <th />
-            </tr>
-          </thead>
-          <tbody>
-            {users.map((user, idx) => {
-              return renderUserRow(user, idx);
-            })}
-            {invites.map((user, idx) => {
-              return renderUserRow(user, idx, { invite: true });
-            })}
-          </tbody>
-        </table>
-      </div>
-    );
-  }
-
   render () {
-    const { renderModal, renderSmtpWarning, renderUserTable, toggleInviteUserModal } = this;
-    const { config, loadingInvites, loadingUsers, users, invites } = this.props;
-    const resourcesCount = users.length + invites.length;
-    if (loadingInvites || loadingUsers) {
-      return false;
-    }
+    const { renderModal, renderSmtpWarning, toggleCreateUserModal, onSearchQueryChange } = this;
+    const { config } = this.props;
+    const { searchQuery } = this.state;
 
     return (
       <div className={`${baseClass} body-wrap`}>
-        <p className={`${baseClass}__page-description`}>Invite new users, customize user permissions, and disable users in Fleet.</p>
+        <p className={`${baseClass}__page-description`}>Create new users, customize user permissions, and remove users from Fleet.</p>
         {renderSmtpWarning()}
-        <div className={`${baseClass}__add-user-wrap`}>
-          <p className={`${baseClass}__user-count`}>{resourcesCount} users</p>
+        {/* TODO: find a way to move these controls into the table component */}
```

Comment:
Are you thinking the table component could receive some other components to render as props?

Reply:
just thinking if we want to include button and input table controlls in the data table. The more I think about it, the more it seems the table needs one more layer of abstraction, like a tableContainer. currently its something like Page => DataTable, but adding one more component like Page => TableContainer => DataTable would allow us to put some of these table controls next to the table data and conditionally show when required.